vm_to_cluster.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
I don't see this referenced anywhere.  Is this supposed to be replaced by vm_to_cluster.rb?

Reply:
I think this is a legacy script which got added back? Will drop again...